### PR TITLE
[IAI] Fix version tag update in IAI deployment.

### PR DESCRIPTION
**Pull request by dacolgit**
Bug: [IAI deployment tags are outdated](https://msazure.visualstudio.com/One/_workitems/edit/12451621)

### CODE REVIEW

File: deploy/src/Microsoft.Azure.IIoT.Deployment/Configuration/HelmSettings.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ class HelmSettings {
         // Defaults
         public static string _defaultRepoUrl = "https://microsoft.github.io/charts/repo";
         public static string _defaultChartVersion = "0.3.2";
-        public static string _defaultImageTag = "2.7.206";
```
Comment (karok2m) [inline deploy/src/Microsoft.Azure.IIoT.Deployment/Configuration/HelmSettings.cs]: Please revert this back to `2.7.206`. Updating IAI for `2.8.1` release entails not only change of image tag and chart version but also cleanup of resources that were deployed for microservices that are removed from the deployment. And we have a separate PBI for doing all of that.

File: deploy/src/Microsoft.Azure.IIoT.Deployment/appsettings.json [hunk 2]
```diff
@@ -112,7 +112,7 @@
     // azure-industrial-iot Helm chart version
     "ChartVersion": "0.3.2",
     // Azure IIoT components image tag
-    "ImageTag": "2.7.206",
```
Comment (karok2m) [inline deploy/src/Microsoft.Azure.IIoT.Deployment/appsettings.json]: Please revert this back to `2.7.206`. Updating IAI for `2.8.1` release entails not only change of image tag and chart version but also cleanup of resources that were deployed for microservices that are removed from the deployment. And we have a separate PBI for doing all of that.

**Review by karok2m — CHANGES_REQUESTED** [review]
Please revert changes for default image tag version. That change along with change of hem chart version and cleanup of resources that are no longer needed will be tackled in a separate PR. Everything else looks good.

File: deploy/src/Microsoft.Azure.IIoT.Deployment/appsettings.json (hunk 2 above)
Comment (dacolgit) [reply deploy/src/Microsoft.Azure.IIoT.Deployment/appsettings.json]: reverted

File: deploy/src/Microsoft.Azure.IIoT.Deployment/Configuration/HelmSettings.cs (hunk 1 above)
Comment (dacolgit) [reply deploy/src/Microsoft.Azure.IIoT.Deployment/Configuration/HelmSettings.cs]: reverted

**Review by karok2m — APPROVED** [review]

**Pull request merged**